espanol/poder-pegaso/poder-pegaso.md

```diff
@@ -0,0 +1 @@
+El pegaso hizo uso de sus fuertes alas y detuvo el avance de los invasores. Sin embargo, luego de unos minutos el pegaso estaba cansado y los invasores se aprovecharon de esto para avanzar y estar frente a frente al pegaso. Es entonces, que el pegaso ...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.